src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.h

```diff
@@ -109,7 +109,8 @@ namespace winrt::PowerToys::PowerAccentKeyboardService::implementation
                                                                LetterKey::VK_MINUS,
                                                                LetterKey::VK_SLASH_,
                                                                LetterKey::VK_DIVIDE_,
-                                                               LetterKey::VK_MULTIPLY_, };
+                                                               LetterKey::VK_MULTIPLY_,
+                                                               LetterKey::VK_QUOT_, };
```

Comment:
Thanks for the changes, I think this one can be removed, right?

Reply:
yes